Makefile

```diff
@@ -64,7 +64,7 @@ check: pylint pycodestyle pydocstyle ## Run linters and static analysis
 
 .PHONY: pylint
 pylint: install
-	$(PYLINT) $(PACKAGES) $(CONFIG) --rcfile=.pylint.ini
+	$(PYLINT) $(PACKAGES) $(CONFIG) --rcfile=.pylintrc
```

Comment:
I prefer the `.pylint.ini` filename so that everyone's editor gets syntax highlighting by default.